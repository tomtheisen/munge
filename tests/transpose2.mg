#(
    /.+/ => { _ push(lines) }
    { 
        0 for(lines) { _ len max }      ! max input row length is left on the stack 
        times {                         ! output row iteration
            i set(row) drop
            for(lines) {                ! output column iteration
                _ get(row) skip 1 take  ! get character at index
                1 lpad                  ! replace "" with " "
            }
            "\n"
        }
    }
    /\s+$/m => ""       ! trim line endings
)