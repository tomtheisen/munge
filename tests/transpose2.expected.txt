adhl
beim
cfjn
 l
 o
 l